spatial-lib/test/cmr/spatial/test/arc.clj

```diff
@@ -1,20 +1,21 @@
 (ns cmr.spatial.test.arc
-  (:require [clojure.test :refer :all]
-            ; [clojure.test.check.clojure-test :refer [defspec]]
-            ;; Temporarily included to use the fixed defspec. Remove once issue is fixed.
-            [cmr.common.test.test-check-ext :refer [defspec]]
-
-            [clojure.test.check.properties :refer [for-all]]
-            [clojure.test.check.generators :as gen]
-
-            ;; my code
-            [cmr.spatial.math :refer :all]
-            [cmr.spatial.arc :as a]
-            [cmr.spatial.point :as p]
-            [cmr.spatial.vector :as v]
-            [cmr.spatial.mbr :as mbr]
-            [cmr.spatial.test.generators :as sgen]))
-
+  (:require
+   [clojure.test :refer [are deftest is testing]]
+   ; [clojure.test.check.clojure-test :refer [defspec]]
+   ;; Temporarily included to use the fixed defspec. Remove once issue is fixed.
+   [cmr.common.test.test-check-ext :refer [defspec]]
+
+   [clojure.test.check.properties :refer [for-all]]
+
+   ;; my code
```

Comment:
I think we can remove this comment

Reply:
I also think these are strange, lots of files have them. I think the author was trying to say the include above are boiler plate things to make tests work, where as the bottom things are application specific. I can just mix them all up or rename this. What do you think?